examples/tensorflow/real-time-object-detection-rpi4/README.md

```diff
@@ -0,0 +1,39 @@
+# Real-time Object Detection on Raspberry Pi 4
+
+## Requirements
+
+- Raspberry Pi 4 with Raspberry Pi OS (64-bit)
+- Python 3.11
```

Comment:
Presumably this would work on a Pi 5 as well? That would be something to test, though I appreciate you probably don't have one. Well, I can try it once everything else is sorted out!

Reply:
I noticed that the TensorFlow example code in the 'picamera 2' repository doesn't function properly on a Raspberry Pi 4 with Raspberry Pi OS (64-bit). In response, I've created my own version of the code and submitted this Pull Request to assist developers who may encounter this configuration in the future.

As for testing on Raspberry Pi 5, unfortunately, I currently don't have a Pi 5 card available. However, I'm committed to testing the code on a Raspberry Pi 5 as soon as I acquire one.